src/nodedef.cpp

```diff
@@ -169,40 +169,75 @@ void NodeBox::deSerialize(std::istream &is)
 	TileDef
 */
 
+#define TILE_FLAG_BACKFACE_CULLING 0x0001
```

Comment:
defining these as `(1 << 0)`, `(1 << 1)`, ... is a little nicer
just my personal preference

Reply:
ok